integration-tests/integration/allocateKeyWorker.spec.js

```diff
@@ -157,7 +157,7 @@ context('Allocate key worker to unallocated prisoners', () => {
             .find('a')
             .contains('Alff, Ferinand')
             .should('have.attr', 'href')
-            .should('include', 'http://localhost:3002/prisoner/ABC123')
+            .should('include', '/prisoner/ABC123')
           expect(offenders[0].prisonNo).to.eq('ABC123')
```

Comment:
I've removed the port and host from these because at least locally they are spinning up against a different port and I'm not sure it add much - but can put them back in if required.